pkgs/build-support/cc-wrapper/default.nix

```diff
@@ -17,6 +17,7 @@
 , isGNU ? false, isClang ? cc.isClang or false, gnugrep ? null
 , buildPackages ? {}
 , libcxx ? null
+, isCompilerRT ? false
```

Comment:
Isn't `useLLVM` the only way to get a compiler-rt `stdenv.cc` at the moment? `stdenv.hostPlatform.useLLVM or false` should also be a feasible replacement.

This disambiguator is probably useful regardless, but I'm wondering if we should be conservative about what we tack on here and if there's possibly a better system for this?

Reply:
No, you can use this with `llvmPackages.tools.clangUseLLVM` etc.

Probably there is a better system. e.g. something like `compiler-rt` is perhaps plausibly an attribute of the platform (if it's required to be consistent across libraries and executables being linked together) whereas something like the linker really isn't (at least on platforms where there are multiple interchangeable linkers). But I'd prefer to separate out these improvements to how these are handled from the functional addition in this PR, unless there's a better current alternative.

Reply:
@sternenseemann yeah I think this is good too. `useLLVM` is kind of a sledge hammer when we ought to support more mix and match. I would not be opposed to specifying the *default* like we do for the linker in the platform description, but non-default compilers can do different things so we can track in the wrapper too.